.mailmap

```diff
@@ -12,3 +12,4 @@
 # Real Name <email@address> nickname
 # Real Name <email@address> Name, Real
 Jeffrey Kelling <j.kelling@hzdr.de> jkelling
+David Pape <d.pape@hzdr.de> David
```

Comment:
🙈 Sorry. Additionally, I changed my name now 😄

Reply:
No worries. I'll rather see this file as a record of contributions or similar... ;)